test/system/150-login.bats

```diff
@@ -398,6 +398,84 @@ EOF
     assert "$output" =~ "--retry-delay .*pull failures \(default \"5s\"\)"
 }
 
+@test "push and pull zstd chunked image" {
+    iid=localhost:${PODMAN_LOGIN_REGISTRY_PORT}/$(random_string 10 | tr A-Z a-z)
+
+    run_podman push \
+               --compression-format zstd:chunked \
+               --cert-dir ${PODMAN_LOGIN_WORKDIR}/trusted-registry-cert-dir \
+               --creds ${PODMAN_LOGIN_USER}:${PODMAN_LOGIN_PASS} \
```

Comment:
These two lines are a great candidate for refactoring into `$pushpullargs` or somesuch